src/lib/storage/table.cpp

```diff
@@ -460,11 +459,9 @@ void Table::add_soft_order_constraint(const TableOrderConstraint& table_order_co
     Assert(column_id < column_count, "ColumnID out of range.");
   }
 
-  {
-    const auto scoped_lock = acquire_append_mutex();
+    const auto append_lock = acquire_append_mutex();
```

Comment:
Indent ... I guess you are lazy and let format do the entire work here? Tzzz ....

Reply:
Of course I am. Yes, did not format yet since this was still WIP.
Life's too short to indent manually, I guess.